raiden/storage/sqlite.py

```diff
@@ -454,6 +466,20 @@ def _query_events(self, limit: int = None, offset: int = None):
 
         return cursor.fetchall()
 
+    def _query_state_changes(self, limit: int = None, offset: int = None):
+        limit, offset = _sanitize_limit_and_offset(limit, offset)
+        cursor = self.conn.cursor()
+
+        cursor.execute(
+            '''
+            SELECT data, log_time FROM state_changes
+                ORDER BY identifier ASC LIMIT ? OFFSET ?
+            ''',
+            (limit, offset),
+        )
+
+        return cursor.fetchall()
```

Comment:
`SQLiteStorage` has `_get_state_changes` which is does exactly what you need. You could have `get_state_changes` that you declared below call that function and remove this one. Unless there's a specific reason you introduced this function, in this case, i would update `_get_state_changes` to accomodate your needs instead of introducing yet another state-changes-querying-function.